apps/cli/src/diagnostic/DebugLlmReq.ts

```diff
@@ -1,28 +1,27 @@
 import { Timer } from "./Timer";
 
-import { ChatMessageLog, ChatMessage } from "@evo-ninja/agent-utils";
+import { ChatLogs, ChatMessage } from "@evo-ninja/agent-utils";
 
 export class DebugLlmReq {
   constructor(
     public time: Timer,
-    public chat: ChatMessageLog,
+    public chatLogs: ChatLogs,
     public response?: ChatMessage
   ) { }
 
   get tokens() {
-    return this.chat["persistent"].tokens +
-      this.chat["temporary"].tokens;
+    return this.chatLogs.tokens;
   }
 
   toString(): string {
     return JSON.stringify(this.toJSON(), null, 2);
   }
 
-  toJSON() {
+  toJSON(): unknown {
```

Comment:
why are we returning `unknown` if we know the structure that we're returning?

Reply:
Just pushed an update with the return types updated.